wps-cloudformation-template.yaml

```diff
@@ -77,21 +77,23 @@ Mappings:
       BucketName: "aws-wps-production"
     systest:
       BucketName: "aws-wps-systest"
+    sandbox:
+      BucketName: "aws-wps-sandbox"
 Conditions:
-  CreateProdSystestBucket: !And
-    - !Equals [!Ref createProdSystestBucket, true]
-    - !Not [!Condition CreateSandboxDevBucket]
-  CreateSandboxDevBucket: !Not [!Or [!Equals [ !Ref environment, production ], !Equals [ !Ref environment, systest ]]]
+  CreateEphemeralBucket: !Equals [ !Ref environment, dev ]
+  CreateBucket: !And
```

Comment:
"CreatePersistentBucket" would be good for disambiguation and future readability

Reply:
Done